nav2_collision_monitor/include/nav2_collision_monitor/types.hpp

```diff
@@ -29,7 +31,15 @@ struct Velocity
   {
     const double first_vel = x * x + y * y;
     const double second_vel = second.x * second.x + second.y * second.y;
-    return first_vel < second_vel;
+    if (second_vel > 0.0) {
```

Comment:
I'd make this some small epsilon like 1e-3 for small odometric error for not quite pure rotations but largely pure rotations

Reply:
No need, since latest [comment](https://github.com/ros-planning/navigation2/pull/3177#issuecomment-1239644594) was met.